components/CaseStudyCard.js

```diff
@@ -0,0 +1,29 @@
+import Paragraph from './typography/Paragraph';
+
+export default function CaseStudyCard({ 
+  studies = []
+}) {
+    if(studies.length === 0){
+      return null;
+    }
+    return (
+      <div className="pt-10 grid lg:grid-cols-3 lg:gap-8 lg:text-center">
```

Comment:
You specified the columns for the grid in the desktop view but No CSS properties to handle it in the mobile view.

Reply:
thanks, but I have no idea what that means 😆 sadly 😞 😆